hooks.js

```diff
@@ -1,7 +1,7 @@
 export const hooks = [
   {
     name: "useArray",
-    author: 'kitze',
+    author: "kitze",
```

Comment:
keep it as single quotes only

Reply:
These got changed during `formatOnSave` with vscode.
will change it but do you have plan of adding linters ?